package.json

```diff
@@ -23,8 +23,8 @@
   "scripts": {
     "devtools": "node source/resource/qx/tool/bin/build-devtools",
     "website": "node source/resource/qx/tool/bin/build-website",
-    "pretest": "node bootstrap-compiler",
-    "test": "node ./bin/build/qx lint --warnAsError && node ./bin/build/qx --block-global-framework test"
+    "pretest": "node bootstrap-compiler --target=build",
+    "test": "node ./bin/build/qx test --fail-fast --block-global-framework 2>nul | npx tap-summary"
```

Comment:
`2>nul` looks like a Windowsism. On Linux, that would be `2>/dev/null` (unless you wanted to write stderr to a file named `nul`). Does this code need to run operating system independent?

Reply:
And how to redirect stderr to stout?
Yes it should be  operating system independent.

Reply:
found it. Same as in windows. 2>&1